subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -121,8 +121,6 @@ Gradle provides a fined-grained view of the actual work being performed directly
 </figure>
 +++++
 
-CAUTION: Some work executed in parallel in not displayed yet e.g. parallel test execution. Future versions of Gradle will improve the granularity of the work in progress.
-
 The following portion of the console output shows the work in-progress display with 8 concurrent workers:
 
```

Comment:
:x: This change only shows tests run by JUnit or TestNG, is that right? I believe we should document that. For example, it does not seem to work with kotlin-test.